### PR TITLE
moves css rules for activityaction.css

### PR DESCRIPTION
**Pull request by dmartzol**
Ref: https://github.com/microsoft/vscode/issues/165169

Moves css rules for ` src/vs/workbench/browser/parts/activitybar/activitybarActions.ts`

### CODE REVIEW

File: src/vs/workbench/browser/parts/activitybar/media/activityaction.css [hunk 1]
```diff
@@ -200,3 +200,93 @@
...
+	background-color: var(--vscode-activityBar-foreground) !important;
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item.active .action-label.codicon,
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item:focus .action-label.codicon,
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item:hover .action-label.codicon {
+	color: var(--vscode-activityBar-foreground) !important;
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item .action-label.profile-activity-item,
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/activitybar/media/activityaction.css:218]: Same rule repeated 3 times, maybe @sandy081 did a mistake here?

File: src/vs/workbench/browser/parts/activitybar/media/activityaction.css [hunk 2]
```diff
@@ -200,3 +200,93 @@
...
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item:hover .action-label.profile-activity-item {
+	color: var(--vscode-activityBarItem-profilesHoverForeground) !important;
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item.checked .active-item-indicator:before {
+	border-left-color: var(--vscode-activityBar-activeBorder);
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item.checked:focus::before {
+	visibility: hidden;
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/activitybar/media/activityaction.css:235]: This was previously only being set if there is a `ACTIVITY_BAR_ACTIVE_FOCUS_BORDER` color, so I think this change changes the semantic in that it is now always `hidden` even without a `ACTIVITY_BAR_ACTIVE_FOCUS_BORDER`

File: src/vs/workbench/browser/parts/activitybar/media/activityaction.css [hunk 3]
```diff
@@ -200,3 +200,93 @@
...
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item.checked:focus .active-item-indicator:before {
+	visibility: visible;
+	border-left-color: var(--vscode-activityBar-activeFocusBorder);
+}
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item.checked .active-item-indicator {
+	z-index: 0;
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/activitybar/media/activityaction.css:244]: Similar to above, this was only set when `ACTIVITY_BAR_ACTIVE_BACKGROUND` was configured.

File: src/vs/workbench/browser/parts/activitybar/media/activityaction.css [hunk 4]
```diff
@@ -200,3 +200,93 @@
...
+	background-color: var(--vscode-activityBar-activeBackground);
+}
+
+/* Styling with Outline color (e.g. high contrast theme) */
+
+.monaco-workbench .activitybar > .content :not(.monaco-menu) > .monaco-action-bar .action-item:before {
+
+	content: "";
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/activitybar/media/activityaction.css:252]: Similar to above, this was behind a check for `activeContrastBorder`

**Review by bpasero — CHANGES_REQUESTED** [review]

**Comment by bpasero** [comment]
We discussed this in the team today: changes to other properties that are not colors or borders will still require some JS code, for example:
* listen to theme changes manually
* toggle a CSS class based on the presence of a color
* in CSS set the non-CSS-variable based properties based on that

**Comment by sandy081** [comment]
This is not a straight forward porting from JS to CSS using variables. This needs some logic to be retained in JS and shall be tested with all possible cases. Sounds not trivial.

**Pull request closed without merging**